error-handling/src/main/java/com/palantir/remoting/http/errors/SerializableError.java

```diff
@@ -35,13 +37,28 @@
 
     public abstract String getMessage();
 
-    public abstract Class<? extends Exception> getExceptionClass();
+    @JsonProperty("exceptionClass")
+    public abstract String getExceptionClassName();
```

Comment:
I suspect the server code should use the String-based mechanism, see http-servers:JsonExceptionMapper.java